theme/snippets/collection-links.liquid

```diff
@@ -0,0 +1,10 @@
+{% assign homepage_links_intro_text = shop.metafields.accentuate.homepage_links_intro_text %}
+
+<div class="CollectionLinks site-padding-x serif-lg mt1_5">
+  {{ homepage_links_intro_text }}
+  {% assign selected_collections = homepage_collection_links | split: '|' %}
+  {% for selected_collection in selected_collections %}
+    {% if forloop.last and forloop.length > 1 %}or{% endif %}
+    <a class="CollectionLinks__button button--style-rounded-border-xxs mx_25" href="{{ collections[selected_collection].url }}" aria-label={{ 'snippets.collection-links.link.aria-label' | t }}>{{ collections[selected_collection].title }}</a>
+  {% endfor %}
```

Comment:
If there are links, loop through them and render "or" in front of the link if it's the last item in the for loop (and not a single link).